CairoMakie/src/overrides.jl

```diff
@@ -115,7 +115,7 @@ draw_poly(scene::Scene, screen::Screen, poly, circle::Circle) = draw_poly(scene,
 function draw_poly(scene::Scene, screen::Screen, poly, polygons::AbstractArray{<:Polygon})
     model = poly.model[]
     space = to_value(get(poly, :space, :data))
-    projected_polys = project_polygon.(Ref(scene), space, polygons, Ref(model))
+    projected_polys = project_polygon.(Ref(poly), space, polygons, Ref(model))
```

Comment:
The `project_x` functions in CairoMakie are currently build to take `transform_func` from passed scenes, which isn't correct but usually irrelevant. However for `PolarAxis` I added a poly without transformations to fill the outer background which is plotted incorrectly with this approach, so I did a quick and dirty fix here (and in utils.jl). Proper fix/cleanup is in #3040.